examples/custom_mouse_functions.py

```diff
@@ -16,6 +16,10 @@
     labeled = ndi.label(blobs)[0]
     labels_layer = viewer.add_labels(labeled, name='blob ID')
 
+    @viewer.mouse_press_callbacks.append
+    def get_event(viewer, event):
+        print(event)
+
```

Comment:
This needs to go! 😂

Reply:
No I take it back, I just saw it's in the examples 🤦 Clearly I need an afternoon ☕